assets/webconfig/js/hyperion.js

```diff
@@ -162,10 +165,57 @@ function sendToHyperion(command, subcommand, msg)
 // -----------------------------------------------------------
 // wrapped server commands
 
-// also used for watchdog
+function requestAuthorization()
+{
+	sendToHyperion("authorize","login",'"username": "Hyperion", "password": "hyperion"');
```

Comment:
I would like like to start a discussion on this: 

I think it is not good practice to give the webui a fixed password which is visible and transmitted in clear text. With this you can fake the webui. I would suggest something like this: 

1. On default after install you can login to webui without an account
2. If you like you can activate logging in with an password on webui frontend by providing an own password which is transmitted and stored in db. 
3. After this you have to login if you startup the webui and the stored pw is used to authenticate through the api. Maybe we can use a cookie for this with a session id which acts as a token
4. We should implement a SMALL binary on server side to either activate, deactivate or change pw. This is useful if you forgot your password. Or if it is possible to set it on webui after initial install someone could "steal" your webui by setting a password you don't know. We should use an extra binary to separate this from running Hyperiond service. 

I would be more than happy to assist with this as I think this would be a very good feature. Especially for security.

Reply:
Edge note: The password and username thing isn't fully implemented. Currently it's useless, as you can't change it anyway :)

Reply:
Yes, I noticed that the username / password for the webui is initially set in the db which is delivered with compiling so to say. 
So you just can check if the combination is right but you can't create a new user or change on at the moment with hyperion?

Reply:
Yes, no logic, no API, no UI :)
I never finished it.

Reply:
Ok, i see. So we would have to do some work there before ;)